presto-docs/src/main/sphinx/release/release-0.250.rst

```diff
@@ -0,0 +1,24 @@
+=============
+Release 0.250
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a memory leak in `NodeTaskMap` which could lead to Full GC or OOMs in coordinator.
+* Improve Dictionary Block `getRegionLogicalSizeInBytes` performance.
```

Comment:
what is the end impact of this change ? This line by itself is not conveying the impact

```
Improve ``DictionaryBlock`` performance. (:pr:`12345`)
```

Reply:
I think it is not all of DictionaryBlock but just one function in it. Going through the description in the pr
"Before this change, abandoning a dictionary with 10 million keys used to 150 times slower than the direct encoding. Now it is only 50 times worse (similar to dictionary encoding)." - So that particular function getRegionLogicalSizeInBytes seems to have a 3x improvement in some cases.

Reply:
Right - but I doubt how many people will benefit from knowing the exact details and function name. I am happy with completely removing this as well.